### PR TITLE
Update ChickensJeiPlugin.java

### PR DESCRIPTION
**Pull request by GenDeathrow**
This will prevent jei crash is a parent Is null or remvoved in configs written on moble untested

### CODE REVIEW

File: src/main/java/com/setycz/chickens/jei/ChickensJeiPlugin.java [hunk 1]
```diff
@@ -97,7 +97,7 @@ public void onRuntimeAvailable(IJeiRuntime jeiRuntime) {
     private List<BreedingRecipeWrapper> getBreedingRecipes() {
         List<BreedingRecipeWrapper> result = new ArrayList<BreedingRecipeWrapper>();
         for (ChickensRegistryItem chicken : ChickensRegistry.getItems()) {
-            if (chicken.getTier() > 1) {
+            if (chicken.getTier() > 1 && chicken.getParent1 != null && chicken.getParent2 != null) {
```
Comment (setycz) [inline src/main/java/com/setycz/chickens/jei/ChickensJeiPlugin.java:100]: [getTier](https://github.com/setycz/ChickensMod/blob/1.10.2/src/main/java/com/setycz/chickens/ChickensRegistryItem.java#L96) already checks for parents not being null. How could this fix any issue

Comment (setycz) [inline src/main/java/com/setycz/chickens/jei/ChickensJeiPlugin.java:100]: You forgot parenthesis.

**Comment by GenDeathrow** [comment]
Yeah I was writing it on my phone and my phone is dumb.. I had overriden thr getTier to return an higher tier chickenm. I was creating a higher tier starting chicken. It works fine for the rest of your code I just didnt realize it would break the jei intergration.  You could even add a has parents method in the registryitem.

**Comment by GenDeathrow** [comment]
https://github.com/GenDeathrow/MoreChickens/blob/master/src/main/java/com/gendeathrow/morechickens/core/ChickensMore.java#L217-L232

This is what I had done, I since took the curse download down because of the issue.  I was creating two new higher tier chickens so I  didn't have to calculate their children tiers and their drop times as I add them in. 

Its just a silly add-on I was doing. I thought I had check all the code to make sure their were no conflicts. I just didn't see this.  I realize this was me issue. and I may have another way of handling it.  

Edit: Just so you know I will respect any decision you make anyways. I will always work around what your doing.

**Comment by setycz** [comment]
Hey, I really like your idea to add `hasParents` method to the `ChickenRegistryItem` and you should be able to overwrite it. I'll add it asap. Thank you for explanation and for the link to the code 👍 , I like to understand reasons of issues so I can think about them in my designs and learn from them.

**Pull request closed without merging**